Update types of standards (#7322)
ethereum/ethereum-org-website: src/content/developers/docs/standards/index.md

## Types of standards {#types-of-standards}

> There are 3 types of EIPs:

Reviewer: ```suggestion
There are 3 types of EIPs:

```
Markdown sometimes doesn't like a list right after text, so adding a new line is helpful here.

> - [Standard Track](https://eips.ethereum.org/)

Reviewer: ```suggestion
- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
```

> - [Meta Track](https://eips.ethereum.org/meta)

Reviewer: ```suggestion
- [Meta Track](https://eips.ethereum.org/meta): describes a process surrounding Ethereum or proposes a change to a process
```

> - [Informational Track](https://eips.ethereum.org/informational)

Reviewer: ```suggestion
- [Informational Track](https://eips.ethereum.org/informational): describes an Ethereum design issue or provides general guidelines or information to the Ethereum community
```

> Further, the Standard Track is sub-divided into 4 categories:

Reviewer: ```suggestion
Furthermore, the Standard Track is subdivided into 4 categories:
```
Nit.

> - [Core](https://eips.ethereum.org/core)

Reviewer: ```suggestion
- [Core](https://eips.ethereum.org/core): improvements requiring a consensus fork
```

> - [Networking](https://eips.ethereum.org/networking)

Reviewer: ```suggestion
- [Networking](https://eips.ethereum.org/networking): improvements around devp2p and Light Ethereum Subprotocol, as well as proposed improvements to network protocol specifications of whisper and swarm.
```

> - [Interface](https://eips.ethereum.org/interface)

Reviewer: ```suggestion
- [Interface](https://eips.ethereum.org/interface): improvements around client API/RPC specifications and standards, and certain language-level standards like method names and contract ABIs.
```

> - [ERC](https://eips.ethereum.org/erc)

Reviewer: ```suggestion
- [ERC](https://eips.ethereum.org/erc): application-level standards and conventions
```

> The main difference in the above categories of Standards Track EIP is where it's deployed. For instance, ERCs are deployed at the application level and therefore don't need to be adopted by all participants. However, Core Track EIP is deployed at the protocol level and requires a broader consensus in the community as all core EIPs must be widely adopted (i.e. all the nodes must upgrade to remain part of the network)

Reviewer: ```suggestion
More detailed information on these different types and categories can be found in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#eip-types)
```

## Types of standards {#types-of-standards}

There are 3 types of EIPs:

> - [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations

Reviewer 2: I will rather call it "EIPs website".
IMHO, Pointing "Standard Track" to  https://eips.ethereum.org/ may not be appropriate.

Reviewer: Yea, that's definitely a typo. Fixing

> - [Meta Track](https://eips.ethereum.org/meta): describes a process surrounding Ethereum or proposes a change to a process

Reviewer 2: "Meta Track" isn't commonly used. Probably "Meta EIPs" will be a better fit.

Reviewer: This is the technical term and is the correct term to use here.

> - [Informational Track](https://eips.ethereum.org/informational): describes an Ethereum design issue or provides general guidelines or information to the Ethereum community

Reviewer 2: Same as comment above. IMO, "Informational EIPs" will be better.

Reviewer: Likewise, this is the correct term to use here.
